packages/fuels-test-helpers/src/node.rs

```diff
@@ -60,6 +60,7 @@ pub struct Config {
     pub block_production: Trigger,
     pub vm_backtrace: bool,
     pub silent: bool,
+    pub chain_conf: ChainConfig,
```

Comment:
call it `chain_config` I think, we are not using `conf` anywhere in the codebase.

Reply:
This was added for testing purposes. I don't think it will be necessary.

Reply:
Reminder to remove it